pkg/test/framework/suite.go

```diff
@@ -107,6 +108,37 @@ func (s *Suite) RequireEnvironment(name environment.Name) *Suite {
 	return s
 }
 
+func compareKubernetesVersion(kuberetesVersion string, minimumVersion int) bool {
```

Comment:
why use int for version? It assumes the caller knows the conversion rule as "1.12" -> 12. This seems fragile.

Reply:
as opposed i saw below accessor already returns typed struct `version.info`.

Reply:
I think its simpler this way - Kubernetes is effectively always on version 1.x like Istio. The alternative would be a user pass `"1.12"`, but I don' think passing the full `version.Info` makes sense

Reply:
using 12 is not future proof.

> kubernetes is effective always on version 1.x

You never know when k8s change to 2.x. Just like y2k problem. Using "1.12" rather than more typed struct is fine. Requiring code developer to assume trim the first major release part is also not good for readability.